cedar-policy-validator/src/schema/namespace_def.rs

```diff
@@ -170,26 +170,26 @@ impl ValidatorNamespaceDef {
 /// map) are fully qualified, and inside the [`SchemaType`]s (values in the
 /// map), all entity/common type references are also fully qualified.
 #[derive(Debug)]
-pub struct TypeDefs {
-    pub(super) type_defs: HashMap<Name, SchemaType<Name>>,
+pub struct CommonTypeDefs {
+    pub(super) defs: HashMap<Name, SchemaType<Name>>,
 }
 
-impl TypeDefs {
-    /// Construct a [`TypeDefs`] by converting the structures used by the schema
-    /// format to those used internally by the validator.
+impl CommonTypeDefs {
+    /// Construct a [`CommonTypeDefs`] by converting the structures used by the
+    /// schema format to those used internally by the validator.
     pub(crate) fn from_raw_typedefs(
```

Comment:
`from_raw_common_type_defs`, or maybe just `from_raw_type_defs` since "common type" is clear from context

Reply:
`from_raw_common_types` since it's taking `namespace_def.common_types`?

Reply:
sounds good